oval/redhat/redhat_test.go

```diff
@@ -0,0 +1,146 @@
+package redhat
```

Comment:
Is this supposed to be a file with integration tests? If so we should start the habit of using build tags `// +build integration` to avoid long running tests.

Reply:
This file is for unit testing. (though it uses httptest...). At least, this test finishes in 0.1 sec.

```
ok      github.com/aquasecurity/vuln-list-update/oval/redhat    0.039s
```

Reply:
I think we don't need a tag yet.